tests/test_twisted.py

```diff
@@ -201,11 +202,14 @@ def test_fakeDHParameterSupport(self, monkeypatch, allFile, recwarn):
         assert ctxFactory.ctxFactory is fakeCtxFactory
         assert "dhParameters" not in recorder.calls[0].kwargs
 
-    def test_realDHParameterSupport(self, monkeypatch, allFile):
+    def test_realDHParameterSupport(self, monkeypatch, tmpdir):
         """
         Pass DH parameters directly to CertificateOptions if the installed
```

Comment:
I think it should be explained, that this is a use case of passing DH parameters explicitly vs. being in some file.